CHANGELOG.md

```diff
@@ -2,12 +2,16 @@
 
 ### Added
 - Support for vars in top level ansible watches. ([#2147](https://github.com/operator-framework/operator-sdk/pull/2147))
+- Added `--watch-namespace` flag to `operator-sdk test local <test-dir> --up-local` command. ([#2149](https://github.com/operator-framework/operator-sdk/pull/2149))
 
 ### Changed
+
 - Upgrade minimal Ansible version in the init projects from `2.4` to `2.6`. ([#2107](https://github.com/operator-framework/operator-sdk/pull/2107))
 - Upgrade Kubernetes version from `kubernetes-1.15.4` to `kubernetes-1.16.2`. ([#2145](https://github.com/operator-framework/operator-sdk/pull/2145))
 - Upgrade Helm version from `v2.15.0` to `v2.16.1`. ([#2145](https://github.com/operator-framework/operator-sdk/pull/2145))
 - Upgrade [`controller-runtime`](https://github.com/kubernetes-sigs/controller-runtime) version from `v0.3.0` to [`v0.4.0`](https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.4.0). ([#2145](https://github.com/operator-framework/operator-sdk/pull/2145))
+- **Breaking Change:** Renamed `--namespace` flag as `--watch-namespace` in `operator-sdk up local` command. ([#2149](https://github.com/operator-framework/operator-sdk/pull/2149))
+- **Breaking Change:** Renamed `--namespace` flag as `--deploy-namespace` in `operator-sdk test local <test-dir> --up-local` command. ([#2149](https://github.com/operator-framework/operator-sdk/pull/2149))
 
```

Comment:
IHMO: this explanation is confusing, how to rename the same command for 2 different things? WDYT about improve this one?

Reply:
Maybe:

Change:
> **Breaking Change:** Remove flag  `--namespace` from `operator-sdk up local` and `operator-sdk test local` commands.

Add: 
> Add flag  `--watch-namespace` for `operator-sdk up local` command in order to allow define the operator scope for local tests.
> Add flag  `--deploy-namespace` for `operator-sdk test local` command in order to allow define the namespace where the operator will be deployed when the flag `--up-local` is used.

Reply:
It's a bit more complicated @camilamacedo86 because we broke one flag into two. Maybe we shouldn't have anything in `Added`, and just have two lines in `Changes`
> **Breaking Change:** Renamed `--namespace` flag as `--watch-namespace` in `operator-sdk up local` command. ([#2149](https://github.com/operator-framework/operator-sdk/pull/2149))
> **Breaking Change:** Split `--namespace` flag into `--deploy-namespace` and `--watch-namespace` in `operator-sdk test local` command to decouple the deploy namespace from the watch namespace. ([#2149](https://github.com/operator-framework/operator-sdk/pull/2149))